boards/nucleo-f401/board.c

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright (C) 2014 Freie Universität Berlin
```

Comment:
Copyright? Most files say "2014 fu berlin", is that correct? at least the year is wrong...